### PR TITLE
Clarify the differences between the SelectBox and Lookup widgets

### PR DESCRIPTION
**Pull request by AlexJameson**
image

TO DO: send to correctors, cherry pick to 19.2.

### CODE REVIEW

**Review by onebalaban — COMMENTED** [review]
Looks OK

**Review by RomanTsukanov — APPROVED** [review]

File: api-reference/10 UI Widgets/dxAutocomplete/dxAutocomplete.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ placeholder: "City"
 dx.web.js, dx.viz-web.js, dx.all.js
 
 ---
+[note] **Autocomplete** displays the suggestions from the **dataSource** based on the text field input. If you need extended functionality, such as grouping or paging, consider using [SelectBox](/api-reference/10%20UI%20Widgets/dxSelectBox '/Documentation/ApiReference/UI_Widgets/dxSelectBox/') instead.
```
Comment (San4es) [inline api-reference/10 UI Widgets/dxAutocomplete/dxAutocomplete.md]: >such as grouping

https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxAutocomplete/Configuration/#grouped

**Review by San4es — APPROVED** [review]
LGTM

File: concepts/05 Widgets/Lookup/35 Lookup vs SelectBox.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+The **Lookup** and [SelectBox](/api-reference/10%20UI%20Widgets/dxSelectBox '/Documentation/ApiReference/UI_Widgets/dxSelectBox/') widgets implement similar functionality - both of them consist of a text field and show a collection of items in the drop-down list, but they also have differences:
```
Comment (arminal) [inline concepts/05 Widgets/Lookup/35 Lookup vs SelectBox.md]:
```suggestion
The **Lookup** and [SelectBox](/api-reference/10%20UI%20Widgets/dxSelectBox '/Documentation/ApiReference/UI_Widgets/dxSelectBox/') widgets implement a similar functionality - while both consist of a text field and display a collection of items in the drop-down list, they also have differences:
```

File: concepts/05 Widgets/Lookup/35 Lookup vs SelectBox.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+- **Drop-down field content**   
+The **SelectBox** has only a list of items in a drop-down field whereas **Lookup** also has a search field and the Cancel button.
+
+- **Appearance on desktop and mobile devices**    
+The **SelectBox** looks the same on any platform while **Lookup** changes its appearance. The **Lookup**'s drop-down field is replaced with a popup window that occupies most of the screen. This design optimizes **Lookup** for mobile devices.
```
Comment (arminal) [inline concepts/05 Widgets/Lookup/35 Lookup vs SelectBox.md]:
```suggestion
The **SelectBox** looks the same on any platform, while **Lookup** changes its appearance. The **Lookup**'s drop-down field is replaced with a popup window that occupies most of the screen. This design optimizes **Lookup** for mobile devices.
```

**Pull request merged**